test/Microsoft.Net.Http.Server.FunctionalTests/Utilities.cs

```diff
@@ -74,5 +78,21 @@ internal static WebListener CreateServer(string scheme, string host, int port, s
             listener.Start();
             return listener;
         }
+
+        internal static async Task<RequestContext> AcceptAsync(this WebListener server, TimeSpan timeout)
```

Comment:
Doc comment that this should be used in all tests to prevent un-expected hangs when a request does not arrive.